src/test/java/com/hocs/test/pages/dcu/DCUProgressCase.java

```diff
@@ -314,6 +326,32 @@ public void generateDCUSearchCaseData(String infoValue, String infoType) throws
                 correspondents.addANonMemberCorrespondentOfType("Constituent");
                 correspondents.confirmPrimaryCorrespondent();
                 break;
+            case "ALL":
+                dashboard.selectCreateSingleCaseLinkFromMenuBar();
+                if (!nextButton.isVisible()) {
+                    dashboard.selectCreateSingleCaseLinkFromMenuBar();
+                }
+                createCase.selectCaseType("MIN");
+                clickTheButton("Next");
+                createCase.editReceivedDate("01/01/2022");
+                createCase.storeCorrespondenceReceivedDate();
+                documents.uploadFileOfType("docx");
+                clickTheButton("Create case");
+                confirmationScreens.storeCaseReference();
+                dashboard.goToDashboard();
```

Comment:
same here, why not just call the create case method instead of relisting all these steps?